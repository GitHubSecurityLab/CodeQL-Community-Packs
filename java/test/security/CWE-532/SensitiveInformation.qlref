security/CWE-532/SensitiveInformation.ql
