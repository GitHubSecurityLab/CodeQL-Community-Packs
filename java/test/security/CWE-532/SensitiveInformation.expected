edges
| SensitiveInformation.java:17:23:17:62 | (...)... : String | SensitiveInformation.java:18:26:18:55 | ... + ... |
| SensitiveInformation.java:17:23:17:62 | (...)... : String | SensitiveInformation.java:19:28:19:31 | attr |
| SensitiveInformation.java:17:23:17:62 | (...)... : String | SensitiveInformation.java:20:66:20:69 | attr : String |
| SensitiveInformation.java:17:31:17:62 | getAttribute(...) : Object | SensitiveInformation.java:17:23:17:62 | (...)... : String |
| SensitiveInformation.java:20:31:20:81 | encodeToString(...) : String | SensitiveInformation.java:26:19:26:30 | responseBody |
| SensitiveInformation.java:20:66:20:69 | attr : String | SensitiveInformation.java:20:66:20:80 | getBytes(...) : byte[] |
| SensitiveInformation.java:20:66:20:80 | getBytes(...) : byte[] | SensitiveInformation.java:20:31:20:81 | encodeToString(...) : String |
nodes
| SensitiveInformation.java:17:23:17:62 | (...)... : String | semmle.label | (...)... : String |
| SensitiveInformation.java:17:31:17:62 | getAttribute(...) : Object | semmle.label | getAttribute(...) : Object |
| SensitiveInformation.java:18:26:18:55 | ... + ... | semmle.label | ... + ... |
| SensitiveInformation.java:19:28:19:31 | attr | semmle.label | attr |
| SensitiveInformation.java:20:31:20:81 | encodeToString(...) : String | semmle.label | encodeToString(...) : String |
| SensitiveInformation.java:20:66:20:69 | attr : String | semmle.label | attr : String |
| SensitiveInformation.java:20:66:20:80 | getBytes(...) : byte[] | semmle.label | getBytes(...) : byte[] |
| SensitiveInformation.java:26:19:26:30 | responseBody | semmle.label | responseBody |
subpaths
#select
| SensitiveInformation.java:18:26:18:55 | ... + ... | SensitiveInformation.java:17:31:17:62 | getAttribute(...) : Object | SensitiveInformation.java:18:26:18:55 | ... + ... | Sensative data is being logged $@. | SensitiveInformation.java:17:31:17:62 | getAttribute(...) | user-provided value |
| SensitiveInformation.java:19:28:19:31 | attr | SensitiveInformation.java:17:31:17:62 | getAttribute(...) : Object | SensitiveInformation.java:19:28:19:31 | attr | Sensative data is being logged $@. | SensitiveInformation.java:17:31:17:62 | getAttribute(...) | user-provided value |
| SensitiveInformation.java:26:19:26:30 | responseBody | SensitiveInformation.java:17:31:17:62 | getAttribute(...) : Object | SensitiveInformation.java:26:19:26:30 | responseBody | Sensative data is being logged $@. | SensitiveInformation.java:17:31:17:62 | getAttribute(...) | user-provided value |
